README.md

```diff
@@ -1,8 +1,107 @@
 # ActiveRecord Associations in Sinatra
 
-##TEACHER OUTLINE
-+ Creating multiple tables
-+ Describe has many/belongs to relationships
-+ Setting up has_many, belongs_to relationship between models
-+ Discuss the use of primary keys as the link between models in ActiveRecord
-+ Should allude to other types of relationships (many to many, one to one, etc) but not discuss join tables.
\ No newline at end of file
+## Objectives
+
+1. Understand primary and foreign key.
+2. Understand the has many/ belongs to relationship.
+
+
+### Primary Keys
+
+Let's assume we have two tables in our database `cats` and `owners`, which we created from the command line using rake.
+
+#### How to create a table with ActiveRecord
+
+Example on how to create a cats table from the command line:
+`rake db:create_migration NAME="create_cats"`
+
+This will give us an empty migration in our `db/migrate/` folder. Now lets give our cats table attributes: `name`, `age` and `breed`. This will go into our up method. In the down method we will also tell the migration what should happen, if for any reason we want to rollback our migration. Here, it would be the opposite of `create table` which is `drop table`.
+
+```ruby
```

Comment:
Looks like this comes in the pre-written code with this repo - do you want this to be here, or are you going to have the students build this?